src/resumable-upload.ts

```diff
@@ -769,7 +775,9 @@ export class Upload extends Writable {
     const headers: GaxiosOptions['headers'] = {
       'x-goog-api-client': `${getRuntimeTrackingString()} gccl/${
         packageJson.version
-      } gccl-invocation-id/${this.currentInvocationId.chunk}`,
+      }-${getModuleFormat()} gccl-invocation-id/${
```

Comment:
Not a blocker but it would be easier to cipher through this if the header changes was decoupled in a separate PR